.travis.yml

```diff
@@ -20,7 +20,12 @@ before_install:
 install:
   - git clone https://github.com/iiasa/ixmp.git
   - cd ixmp && python setup.py install && cd ..
-  - pip install .[reporting]
+# Upon the next pyam release (>0.2.0), these two lines should be removed.
+  - git clone https://github.com/IAMconsortium/pyam.git
```

Comment:
@gidden would it be better to use `pip install --quiet --no-cache-dir git+https://github.com/IAMconsortium/pyam.git@master`

Reply:
Good suggestion.